bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/resources/PdfUploader.kt

```diff
@@ -0,0 +1,35 @@
+package net.gini.android.bank.sdk.exampleapp.ui.resources
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiSelector
+
+class PdfUploader {
+    private val device = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
+    fun uploadPdfFromFiles() {
+        //Wait for 'tab'
+        device.waitForIdle()
+        val tabTitle = device.findObject(UiSelector().className("android.widget.TextView").text("Recent"))
+        if (tabTitle.exists() && tabTitle.text == "Recents") {
+            //Interact with hamburger menu
+            val hamburgerMenu = device.findObject(UiSelector().className("android.widget.ImageButton"))
+            hamburgerMenu.click()
```

Comment:
[Clarification] Don't we need to check `exists()` before calling `click()`, like you did for `tabTitle` ?
Same question for the other fields below.